spec/identifiers/doi_spec.rb

```diff
@@ -7,6 +7,12 @@
     expect(described_class.extract(str)).to contain_exactly('10.1049/el.2013.3006')
   end
 
+  it 'extracts DOIs from any where in a string' do
+    str = 'This is an example of DOI - 10.1049/el.2013.3006 - which is entirely valid'
```

Comment:
It was, but there was no test for "in the middle of a string". Since I was tweaking that kind of code, I wanted to double check it continued to work.